infrastructure/zk/src/env.ts

```diff
@@ -64,11 +64,16 @@ export function set(env: string, print: boolean = false) {
 }
 
 // override env with variables from init log
-function loadInit() {
+function loadInit(zksyncEnv: string) {
     if (fs.existsSync('etc/env/.init.env')) {
         const initEnv = dotenv.parse(fs.readFileSync('etc/env/.init.env'));
         for (const envVar in initEnv) {
             process.env[envVar] = initEnv[envVar];
+            if (zksyncEnv.startsWith('ext-node')) {
+                // We want to name env vars used by external nodes idiomatically, hence the added `EN_` prefix.
+                // The original names are still used in some integration tests, so we expose the original var as well.
+                process.env[`EN_${envVar}`] = initEnv[envVar];
+            }
```

Comment:
I haven't come up with a good way to copy these variables from `.init.env` (rn, we require the only variable: `CONTRACTS_VALIDATOR_TIMELOCK_ADDR`). Theoretically, the EN could ready this env var directly using `std::env::var`, but IMO it looks too ad-hoc.

Reply:
pinging @Deniallugo for visibility - how will this change in the new CLI?

Reply:
We won't use env variables anymore.

We will just provide files with necessary date. 

but, what to do with tests is a good question